input.js

```diff
@@ -5,13 +5,16 @@ const readline = require('readline')
 const Input = (() => {
 
     const rl = readline.createInterface({
+        completer,
         input: process.stdin,
-        output: process.stdout
+        output: process.stdout,
```

Comment:
What is the benefit of having a trailing comma here? Is this part of our style guide now?